Dockerfile

```
@@ -47,7 +47,17 @@ FROM ${BASE}-base AS base
 ARG KIMAI
 ARG TIMEZONE
 
-LABEL maintainer="tobias@neontribe.co.uk"
+LABEL org.opencontainers.image.title="Kimai" \
+      org.opencontainers.image.description="Kimai is a time-tracking application." \
+      org.opencontainers.image.authors="Kimai Community" \
```

@tobybatch would you prefer

```suggestion
      org.opencontainers.image.authors="Toby Batch and  theKimai Community" \ 
```

??